File: webservice/Frikollection_Api/Frikollection_Api/DTOs/ProductExtension/ProductExtensionDto.cs
namespace Frikollection_Api.DTOs.ProductExtension
{
    public class ProductExtensionDto
    {
        public int? Hp { get; set; }
        public string? PokemonTypes { get; set; }
        public string? EvolvesFrom { get; set; }
        public List<AbilityDto>? Abilities { get; set; }
        public List<AttackDto>? Attacks { get; set; }
        public int? ConvertedRetreatCost { get; set; }
        public string? Package { get; set; }
        public string? Expansion { get; set; }
    }
}

File: mobile_desktop_app/FrikollectionMobileDesktop/composeApp/src/commonMain/kotlin/org/example/frikollection_mobile_desktop/models/ProductExtensionDto.kt
package org.example.frikollection_mobile_desktop.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class ProductExtensionDto(
    val hp: Int? = null,
    val pokemonTypes: String? = null,
    val evolvesFrom: String? = null,
    val abilities: List<AbilityDto>?,
    val attacks: List<AttackDto>?,
    val convertedRetreatCost: Int? = null,
    @SerialName("package")
    val packageName: String? = null,
    val expansion: String? = null
)